src/components/feed/feed.jsx

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+/* 
+ @author [Jenny Doctor](https://github.com/jendoc)
+*/
+
+export const Feed = ({ children }) => {
+  return <div className="bg-alpha" role={"feed"}>{children}</div>;
+};
+
+const Heading = styled.h3`
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 120%;
+  margin: 16px auto 8px 16px;
+  opacity: 0.7;
```

Comment:
I realized in another PR that the `text-white75` class no longer applies the opacity like it used to in bootstrap 4, but I think if you change this line to `color: var(--bs-white75)` then it will apply the opacity